tools/stress-cluster/cluster/kubernetes/stress-test-addons/templates/_util.tpl

```diff
@@ -35,11 +35,15 @@ See https://github.com/Masterminds/sprig/tree/master/docs for template function
 {{- /* Copy scenario name into top level keys of global context */}}
 {{- $_global := index . 0 -}}
 {{- $_scenario := index . 1 -}}
-{{- $resourceGroupName := lower (print $_global.Release.Namespace "-" $_scenario "-" $_global.Release.Name "-" $_global.Release.Revision) -}}
+{{- $resourceGroupName := lower (print $_global.Release.Namespace "-" $_scenario.Scenario "-" $_global.Release.Name "-" $_global.Release.Revision) -}}
 {{- /* Use lowercase alpha characters for maximum azure resource naming compatibility */ -}}
 {{- $uniqueTestId := lower (randAlpha 6) -}}
 {{- /* Create add Stress context to top level keys of global context */}}
-{{- $_stress := dict "Scenario" $_scenario "ResourceGroupName" $resourceGroupName "BaseName" $uniqueTestId -}}
+{{- $_stress := dict "Scenario" $_scenario.Scenario "ResourceGroupName" $resourceGroupName "BaseName" $uniqueTestId -}}
```

Comment:
You can remove this first bit since you are doing a full object merge below.

```suggestion
{{- $_stress := dict "ResourceGroupName" $resourceGroupName "BaseName" $uniqueTestId -}}
```